modules/imaging_browser/php/imagingbrowserrowprovisioner.class.inc

```diff
@@ -37,7 +37,12 @@ class ImagingBrowserRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisi
     function __construct()
     {
         $config = \NDB_Config::singleton();
-
+        $DB     = null;
+        try {
+            $DB = \NDB_Factory::singleton()->database();
```

Comment:
This try/catch should not be here. If an exception is thrown something is seriously wrong enough that it should be propagated.